native/helpers/challenges.js

```diff
@@ -453,4 +453,12 @@ export const challenges = [
   { title: 'Emoji Editor', link: '#' },
   { title: 'Sort Table', link: '#' },
   { title: 'Infinite Scroll', link: '#' },
+  {
+    title: 'Fitness Tracker',
+    link: 'fitness-tracker',
```

Comment:
This link is incorrect in the UI
Either make the link `fitness-tracker/tracker.html`, or rename the `tracker.html` file to `index.html`

Reply:
Also place the challenges with same difficulty together in the challenges list